src/components/nodes/startEvent/startEvent.vue

```diff
@@ -48,13 +50,35 @@ export default {
     return {
       shape: null,
       definition: null,
+      showDropdown: false,
     };
   },
+  computed: {
+    isBaseStartEvent() {
+      return this.node.type === 'processmaker-modeler-start-event';
+    },
+  },
+  methods: {
+    removeDropdownOnUnhighlight() {
+      const unwatch = this.$watch('highlighted', highlighted => {
+        if (!highlighted) {
+          this.showDropdown = false;
+          unwatch();
+        }
+      });
```

Comment:
I'm not sure I understand this syntax. Why is there a variable 'unpatch' here? It doesn't appear to be used...

Reply:
From https://vuejs.org/v2/api/#vm-watch:
> `vm.$watch` returns an unwatch function that stops firing the callback:
> ```js
> var unwatch = vm.$watch('a', cb)
> // later, teardown the watcher
> unwatch()
> ```

Except in our case I need to use the unwatch function within the watcher's own callback. It's like a "watch only once" method. Once the dropdown is hidden I didn't want to keep running this watcher as it should only have to set `showDropdown` to `false` once.

I could also keep it set up as a regular watcher and add an early return like:
```js
if (!this.showDropdown) {
  return;
}
```

Reply:
I think I would prefer using a regular watcher with an early return; the way I set it up with unwatch here isn't the greatest.

Reply:
I think a regular watcher would be a little clearer. The current code does have a high 'wtf per minute' count.... ;P

Reply:
With the greatest respect, it seems to be over clever. :)